typings/index.flow.js

```diff
@@ -349,8 +349,21 @@ interface Thenable {
   then: (resolve: () => any, reject?: () => any) => any;
 }
 
+type MapPropsFunction = (
+  props: { [string]: mixed },
+  node: ReactTestRendererJSON
+) => { [string]: mixed };
+
+type FormatOptions = {
+  mapProps?: MapPropsFunction,
+};
+
+type DebugOptions = {
+  message?: string,
+} & FormatOptions;
```

Comment:
`FormatOptions` does not seem to be used anywhere, let's merge it with `DebugOptions` for flow


```suggestion
type DebugOptions = {
  message?: string,
  mapProps?: MapPropsFunction,
};
```